hr_holidays_validator_list/tests/test_hr_leave.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from datetime import datetime
+
+from odoo.exceptions import UserError
+from odoo.tests import TransactionCase
+
+
+class TestHrLeave(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        employee = self.env["hr.employee"].search([])[0]
+        employee.write({"leave_manager_ids": [(6, 0, [6, 2])]})
+        hr_leave_type = self.env["hr.leave.type"].create(
+            {
+                "name": "Leave Test",
+                "request_unit": "hour",
+            }
```

Comment:
```suggestion
            "requires_allocation": "no",
```

Your tests are failing because you create an hr leave with a type that requires to create an allocation beforehand. Add this line to say that it does not require it

Reply:
Hi @victorvermot,  
I have updated the PR. However, the tests are failing in the module `hr_holidays_natural_period`.
Thanks for your time.